README.md

```diff
@@ -101,3 +101,60 @@ If you add a new hook make sure to run it against all files:
 ```shell
 $ pre-commit run --all-files
 ```
+
+<!-- BEGIN_TF_DOCS -->
```

Comment:
this isn't right, the github action and pre-commit hook are looking for a different comment, need to configure one of them to look at a different comment.

Reply:
Please can you provide some more information? Not sure what you mean here.

Reply:
have a look at the README, terradocs is currently there twice. and have a look at the two comments `BEGIN_TF_DOCS` and `BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK`

Reply:
Updated the terraform-docs configuration to match output